src/nvim/ex_getln.c

```diff
@@ -3832,49 +3832,56 @@ static void expand_shellcmd(char_u *filepat, int *num_file, char_u ***file,
   int flags = flagsarg;
   int ret;
 
-  /* for ":set path=" and ":set tags=" halve backslashes for escaped
-   * space */
+  // for ":set path=" and ":set tags=" halve backslashes for escaped
+  // space
   pat = vim_strsave(filepat);
-  for (i = 0; pat[i]; ++i)
-    if (pat[i] == '\\' && pat[i + 1] == ' ')
+  for (i = 0; pat[i]; ++i) {
+    if (pat[i] == '\\' && pat[i + 1] == ' ') {
       STRMOVE(pat + i, pat + i + 1);
+    }
+  }
 
   flags |= EW_FILE | EW_EXEC;
 
-  /* For an absolute name we don't use $PATH. */
-  if (path_is_absolute_path(pat))
+  bool mustfree = false;  // Track memory allocation for *path.
+  // For an absolute name we don't use $PATH.
+  if (path_is_absolute_path(pat)) {
     path = (char_u *)" ";
-  else if ((pat[0] == '.' && (vim_ispathsep(pat[1])
-                              || (pat[1] == '.' && vim_ispathsep(pat[2])))))
+  } else if ((pat[0] == '.' && (vim_ispathsep(pat[1]) 
+        || (pat[1] == '.' && vim_ispathsep(pat[2]))))) {
```

Comment:
This doesn't follow styleguide. `||` should be columned below `vim_ispathsep`, as before.